subsys/mgmt/hawkbit/hawkbit_firmware.c

```diff
@@ -13,7 +13,7 @@ bool hawkbit_get_firmware_version(char *version, int version_len)
 	struct mcuboot_img_header header;
 
 	if (boot_read_bank_header(FIXED_PARTITION_ID(slot0_partition), &header,
-				  version_len) != 0) {
+				  sizeof(struct mcuboot_img_header)) != 0) {
```

Comment:
Probably its better to replace this with `sizeof(header)` instead, so it simply follows the type of the `header` variable and more future proof

Reply:
Yeah, this is actually good point.

Reply:
Is there any reason to not check the compatibility of header version?

https://github.com/zephyrproject-rtos/zephyr/blob/d4c8de514f8b223e8b76c16dbfad482a2ca92edf/subsys/mgmt/updatehub/updatehub_firmware.c#L27-L30

Reply:
`sizeof(header)` is fine for me too.
About the check of the version, this can be added, sure (not the original purpose of the PR but can be done).
Do you want 1 commit for the fix or 2 commits in the history ?

Reply:
@nandojve done!